test/extended/authorization/scc.go

```diff
@@ -75,3 +82,85 @@ var _ = g.Describe("[sig-auth][Feature:SCC][Early]", func() {
 		}
 	})
 })
+
+var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:SCC]", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLIWithPodSecurityLevel("required-scc", psapi.LevelPrivileged)
+
+	g.It("required-scc annotation is being applied to workloads", func() {
+		sccRole, err := oc.AdminKubeClient().RbacV1().ClusterRoles().Create(context.Background(),
+			&rbacv1.ClusterRole{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "required-scc-",
+				},
+				Rules: []rbacv1.PolicyRule{
+					{APIGroups: []string{"security.openshift.io"}, Resources: []string{"securitycontextconstraints"}, Verbs: []string{"use"}},
+				},
+			}, metav1.CreateOptions{},
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		_, err = oc.AdminKubeClient().RbacV1().RoleBindings(oc.Namespace()).Create(context.Background(),
+			&rbacv1.RoleBinding{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "required-scc-",
+					Namespace:    oc.Namespace(),
+				},
+				RoleRef: rbacv1.RoleRef{
+					APIGroup: "rbac.authorization.k8s.io",
+					Kind:     "ClusterRole",
+					Name:     sccRole.Name,
+				},
+				Subjects: []rbacv1.Subject{
+					{
+						Kind:     "User",
+						Name:     oc.Username(),
+						APIGroup: "rbac.authorization.k8s.io",
```

Comment:
Should it be `user.openshift.io`?
```suggestion
						APIGroup: "user.openshift.io",
```